core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorBodyDirective.js

```diff
@@ -28,10 +28,27 @@ oppia.directive('simpleEditorBody', [
         '$scope', 'SimpleEditorManagerService',
         'explorationSaveService', 'explorationRightsService',
         'explorationWarningsService', 'QuestionIdService',
-        function($scope, SimpleEditorManagerService,
+        'StatesToQuestionsService', function($scope, SimpleEditorManagerService,
             explorationSaveService, explorationRightsService,
-            explorationWarningsService, QuestionIdService) {
+            explorationWarningsService, QuestionIdService,
+            StatesToQuestionsService) {
           $scope.data = SimpleEditorManagerService.getData();
+          $scope.chooseQuestionType = false;
+          $scope.modalWrapperVisible = false;
```

Comment:
Yes, that's not a modal but stating @kevinlee12  comment "When I do get a dropdown, I can't close it unless I make a selection, the expected behavior in this case is that I can click elsewhere to close it." and I even think that might have been the intention when the mocks were designed.  Although relevant name according to me in this case must be interactionSelectionBackdropEnable.